modular_skyrat/modules/customization/modules/client/loadout/donator/neck.dm

```diff
@@ -19,3 +19,7 @@
 	name = "Polychromic Cape - Cross"
 	path = /obj/item/clothing/neck/cloak/polychromic/rscloak_cross
 	extra_info = LOADOUT_INFO_THREE_COLORS
+
+/datum/loadout_item/neck/donator/mantle/regal
+	name = "Regal Mantle"
+	path = /obj/item/clothing/neck/mantle/regal
```

Comment:
```suggestion
	path = /obj/item/clothing/neck/mantle/regal
	
```

Reply:
Linters are failing because you don't have a newline here. This code suggestion will fix that

Reply:
set, i think! anything else?